### PR TITLE
Dsresults

### PR DESCRIPTION
**Pull request by sreeder**
This PR adds the functions getdatasetvalues and getdatasetresults. it also adds test and fixes a minor bug in the getsamplingfeaturedatasets function. It also adds all of the requested parameters to the getdatasets function.

### CODE REVIEW

File: odm2api/ODM2/services/readService.py [hunk 1]
```diff
@@ -686,23 +687,109 @@ def getResults(self, ids=None, type=None, uuids=None, actionid=None, simulationi
...
+            q.filter(DataSets.DataSetUUID.in_(uuids))
+        if dstype:
+            q = q.filter(DataSets.DataSetTypeCV == dstype)
+        try:
+            return q.all()
+        except Exception as e:
+            print('Error running Query {}'.format(e))
+        return None
+
+    def getDataSetsValues(self, ids=None, codes=None, uuids=None, dstype=None):
```
Comment (lsetiawan) [inline odm2api/ODM2/services/readService.py]: Where is the docstring for this function?

Comment (sreeder) [reply odm2api/ODM2/services/readService.py]: I knew I forgot something on there! I will add that right now

**Comment by lsetiawan** [comment]
@sreeder Are you merging this or should I? I assume you have tested this? Thanks.

**Comment by sreeder** [comment]
@lsetiawan  yes I have tested it. I even created and uploaded tests for each of the functions. Oh, and I can merge it.

**Pull request merged**